src/pages/postDetail/components/CommentItem.tsx

```diff
@@ -35,29 +37,34 @@ const CommentItem = ({
   commentId,
   type,
   isAnonymous,
+  isNested,
 }: CommentItem) => {
   const { deleteComment } = useDeleteComment(commentId || '', postId || '');
   const { deleteNestedComment } = useDeleteNestedComment(commentId || '', postId || '');
   const [isClick, setIsClick] = useState(false);
   const [isNestedComment, setIsNestedComment] = useState(false);
   const modalRef = useRef<HTMLDivElement>(null);
-
-  const handleBtnClick = () => {
-    setIsClick((prev) => !prev);
+  const commentRef = useRef<HTMLDivElement>(null);
+  const [isBgClick, setIsBgClick] = useState(false);
+
+  const handleDeleteBtn = () => {
+    if (isClick) setIsBgClick(true);
```

Comment:
p5) 여기 클린코드 생각하면 &&로 바꿔도 좋을 듯! 
```suggestion
    isClick && setIsBgClick(true);
```

Reply:
저도 평소에는 그러는 편인데요. 바로밑에 또 if문이 나와서, if문에 두개 있는게, AND연산가 한개,if문 한개 있는거보다 더 가독성이 좋아보이더라고요..?

Reply:
p5) 저도 if문을 사용하는 것이 가독성이 더 좋다고 생각이 들어요!  아래 if문과 통일시킨것도 좋아보이구요!